target/linux/ramips/dts/DIR-506L.dts

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: BSD

```

Comment:
This doesn't seem like a valid identifier. Where did you find it? I can't see it on the
https://spdx.org/licenses/
page and just `BSD` seems really inaccurate.

Reply:
Just write it :)
Replaced to `BSD-2-Clause` - I get in some FreeBSD srcs.